spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisBinderConfigurationProperties.java

```diff
@@ -44,6 +44,8 @@
 
 	private int minShardCount = 1;
 
+	private boolean kplKclEnabled;
```

Comment:
Please, add short descriptive JavaDoc for this field. With period in the end.
This is going to help tooling and docs to generate helpful metadata.

I understand that all other properties are without that, but this is slightly different story which we definitely should fix too. Eventually...